### PR TITLE
2 new documents alongside a gitignore

### PR DESCRIPTION
**Pull request by thomasmarton**

### CODE REVIEW

File: lol/lcuapi/Getting started with the LCU websocket.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+---
+# Getting started with the LCU websocket
+
+In LCU land, rather than the UX process constantly polling the LCU endpoints to see if anything has changed, the LCU API also hosts a websocket allowing the UI to be notified about changes (like you receiving a friend request or a chat message). This guide will show you how you can interact with this websocket to receive notifications that might be needed for your app (like champion select events or winning/losing).
+
+This guide assumes that you have basic knowledge about the LCU API. Check out our other guide to help you get started.
+
+---
+
+The LCU websocket uses the WAMP 1.0 protocol on this websocket. An example implementation in JavaScript can be found [over here](https://gist.github.com/Pupix/eb662b1b784bb704a1390643738a8c15).
```
Comment (Hi-Ray) [inline lol/lcuapi/Getting started with the LCU websocket.md]: Should credit Pupix here as it is his gist.

**Comment by Hi-Ray** [comment]
Overall it seems too much of a guide/tutorial is that the aim of these docs?

**Comment by thomasmarton** [comment]
My vision for the project was to help out the community so they can understand what's available. A knowledgebase style that mixes both documents and tutorials is within this vision in my opinion so I think these 2 documents have a place in hexdocs

File: lol/lcuapi/Getting started with the LCU websocket.md (hunk 1 above)
Comment (thomasmarton) [reply lol/lcuapi/Getting started with the LCU websocket.md]: Pushed a change where he's properly credited

**Pull request merged**